vpr/src/draw/search_bar.cpp

```diff
@@ -96,14 +98,33 @@ void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app) {
             return;
         }
 
-        highlight_blocks((ClusterBlockId)block_id);
+        highlight_cluster_block((ClusterBlockId)block_id);
     }
 
     else if (search_type == "Block Name") {
         std::string block_name = "";
         ss >> block_name;
 
-        highlight_blocks((std::string)block_name);
+        AtomBlockId atom_blk_id = AtomBlockId::INVALID();
+        atom_blk_id = atom_ctx.nlist.find_block(block_name);
+        //If block is found, the CLB containing it is highlighted
+        if (atom_blk_id != AtomBlockId::INVALID()) {
+            ClusterBlockId block_id = atom_ctx.lookup.atom_clb(atom_blk_id);
+            //Highlighting atom block. IF function returns false, highlighting clb that contains
```

Comment:
Comment is a bit confusing.  
contains -> contains it  (I think)
What does this function returning false mean? That's what the comment should explain.